test/scripts/e2e_subs/goal/atomic_abi.py

```diff
@@ -0,0 +1,259 @@
+import json
```

Comment:
WIP. Follow-up #3277 will clean this class up and make it easier to use.

Reply:
Is this test harness really needed for this min-balance in REST PR?